BiconeEvolution/current_antenna_evo_build/XF_Loop/Evolutionary_Loop/Loop_Parts/Part_D/Part_D1_PUEO.sh

```diff
@@ -57,81 +59,25 @@ then
 	mkdir -m775 Run_Outputs/${RunName}/PUEO_Errors
 fi
 
-
-cd Antenna_Performance_Metric
-
-echo "Resuming..."
-cd $PSIMDIR
-
-
-#Let's make sure we're sourcing the correct files
-#source /fs/ess/PAS1960/BiconeEvolutionOSC/new_root/new_root_setup.sh
-#put a copy of this in the repository
-#Probably change this in the future
 source $PSIMDIR/set_env.sh
 
-#If we're doing a real run, we only need to change the setup.txt file once
-#Although we need to be careful, since maybe eventually we'll want to run multiple times at once?
 if [ $DEBUG_MODE -eq 0 ]
 then
-
-	#Shouldn't need this as we input these as paramaters into the simulatePueo call
-	#sed -e "s/Number of neutrinos: 200/Number of neutrinos: ${NNT}/" -e "s/Energy exponent: 20/Energy exponent: $exp/" ${PSIMDIR}/pueoBuilder/components/pueoSim/config/pueo.conf > ${PSIMDIR}/pueoBuilder/components/pueoSim/config/setup.conf
-	
-	# Now we just need to run IceMC from the setup file
-	# Instead of a for loop, we can use a single command
-	# We need the jobs to go form 1 to NPOP*NSEEDS
-	# For the job name, make it the RunName
-	# This will help for directing the output/error files
 	cd $WorkingDir
 	numJobs=$((NPOP*Seeds))
-	maxJobs=252 #for now, maybe make this a variable in the main script
+	maxJobs=252 
+	# Each job will simulate pueosim 40 times for NNT neutrinos each
 	sbatch --array=1-${numJobs}%${maxJobs} --export=ALL,gen=$gen,WorkingDir=$WorkingDir,RunName=$RunName,Seeds=$Seeds,PSIMDIR=$PSIMDIR,NPOP=$NPOP,NNT=$NNT,Exp=$exp --job-name=${RunName} Batch_Jobs/PueoCall_Array.sh
```

Comment:
I just checked the implementation for many runs in the job script. It looks like it'll run 40 PUEOsim runs per job (or whatever `$threads` is set to?). Do we need to change anything about the job array call? Looks like numJobs is still being calculated as `$NPOP*$Seeds`, but I guess it should be `$NPOP*$Seeds/$threads` now?

Reply:
I thought about that, and I think it's fine as is. If we divide by threads here, then we'll just end up multiplying the value of Seeds by a multiple of threads anyways to keep the number of jobs as an integer